app/models/item.rb

```diff
@@ -0,0 +1,10 @@
+class Item < ApplicationRecord
```

Comment:
This Item model looks good, but make sure you also are writing corresponding model tests!